Tests/TokamakCLITests/TestPropsEquatable.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  TestProps.swift
+//  Tokamak
+//
+//  Created by Matvii Hodovaniuk on 4/2/19.
+//
+// TokamakLintTests testPropsIsEquatable test use this file
+
+import Foundation
+
+public protocol Default {
+  var str: String { get }
+}
+
+struct Props: Equatable, Default {
```

Comment:
shouldn't the `Props` type here be a part of a component? The purpose of the check is to verify that only `Props` associated types within a component are `Equatable`